example/index.scss

```
@@ -1,6 +1 @@
-@import "@edx/brand/paragon/fonts";
-@import "@edx/brand/paragon/variables";
-@import "@edx/paragon/scss/core/core";
-@import "@edx/brand/paragon/overrides";
-
 @import "@edx/frontend-component-header/index";
```

```suggestion
@use "@openedx/paragon/dist/core.min.css" as paragonCore;
@use "@openedx/paragon/dist/light.min.css" as paragonLight;

@import "@edx/frontend-component-header/index";
```

For the example app, I think we can use the Paragon installed versions, in this way, it's not necessary to add a js config file, and the styles are applied only for the example app.

---

Corrected at https://github.com/openedx/frontend-component-header/pull/351/commits/4ef75fe0bcd12d84e28a485485297681c58bf2ef